explore/src/main/scala/explore/Routing.scala

```diff
@@ -114,7 +115,7 @@ object Routing {
           ) ~> renderP(
             targetTab
           )
-          | staticRoute("/configurations", ConfigurationsPage) ~> render(UnderConstruction())
+          | staticRoute("/configurations", ConfigurationsPage) ~> render(SequenceEditor())
```

Comment:
So it will be in its own path? I think that's a good idea until we get the proper place for the table